apps/src/javalab/Neighborhood.js

```diff
@@ -16,7 +16,10 @@ const ANIMATED_STEPS = [
 export default class Neighborhood {
   constructor() {
     this.controller = null;
+    // TODO: set this based on level once we have variable sizes
```

Comment:
Could you link this line in the task so it's easy to find when we work on it?

Reply:
done!